DirectX11Proj/PlayerSceneExample.cpp

```diff
@@ -14,29 +14,29 @@ PlayerSceneExample::~PlayerSceneExample()
 {
 }
 
-void PlayerSceneExample::Tick(InputClass* const aInput)
+void PlayerSceneExample::Tick(InputClass* const apInput)
 {
 	//throw std::logic_error("The method or operation is not implemented.");
 	static int x = 0;
 	static int y = 0;
 
 	int mouseRelX = 0;
 	int mouseRelY = 0;
-	aInput->GetMouseRelLocation(mouseRelX, mouseRelY);
+	apInput->GetMouseRelLocation(mouseRelX, mouseRelY);
 
 	x += mouseRelX;
 	y += mouseRelY;
 
 	mCamera->SetRotation(y, x, 0);
 
-	if (aInput->mKeyboardState[DIK_W] & 0x80)
+	if (apInput->mKeyboardState[DIK_W] & 0x80)
```

Comment:
replace 0x80 with an enum in the .h file. Like this:

public:
   enum
   {
      kWFlag = 0x80, // or something
   }

   Class();
   ~Class();